ods_ci/tests/Resources/Files/llm/serving_runtimes/tgis_servingruntime_grpc.yaml

```diff
@@ -18,32 +18,6 @@ spec:
       env:
         - name: TRANSFORMERS_CACHE
           value: /tmp/transformers_cache
```

Comment:
why removed probes ..is it causing the failure for us. I know you might have done due to psap issue but i think having probe could be useful

Reply:
oka